src/main/resources/hudson/plugins/jira/JiraCreateIssueNotifier/help-type.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    <strong>Optional</strong> <br/>
+    This will be the type of the JIRA issue, for project default leave empty.
```

Comment:
In the code I see either name (Bug) is permitted or a numeric value, but this help message doesn't mention this..